lib/templates/bootstrap/select.ios.js

```diff
@@ -55,7 +55,7 @@ class CollapsiblePickerIOS extends React.Component {
             }, animationConfig)).start();
             this.setState({isCollapsed: !this.state.isCollapsed});
           }}>
-          <Text style={pickerValue}>
+          <Text numberOfLines={1} style={pickerValue}>
```

Comment:
@alvaromb 
There is an option `height: 35`, in styles for the `pickerTouchable` style. And if we change the `numberOfLines` prop to value `2` or more we will see the following:
![](https://cl.ly/2B1L0k3l1X3M/Image%202017-04-19%20at%202.55.08%20%D0%9F%D0%9F.png)

I think if someone want to change it he can eject styles and templates (bootstrap) and change it there. Or we can remove `height: 35` from the `pickerTouchable` style but I don't know how it will work.